react-frontend/src/components/StyleComponents/pageContent.js

```diff
@@ -8,6 +8,38 @@ export const GlobaStyleSize = createGlobalStyle`
  }
 `;
 
+export const SimplePageBody = styled.div.attrs({
```

Comment:
I have concerns about all the styles in this file. But perhaps it's easier to address tech debt in here later in a separate issue?